.github/workflows/deployment.yaml

```diff
@@ -0,0 +1,64 @@
+name: Deployment
```

Comment:
I'm a little confused about how we handle multiple environments and deployment logic between them. Would it be worth seeing if we can get a second GCP project so we can make our own `dev` and `prod` environments and work out the associated CD logic? This could totally be another PR.